### PR TITLE
Cgprod 3068 9 slice confirm

### PR DESCRIPTION
**Pull request by FostUK**
Adds nine slice capability to the confirm panel.
Removed uncalled background code.

### CODE REVIEW

File: src/components/shop/menu-buttons.js [hunk 1]
```diff
@@ -43,8 +43,8 @@ const setButtonOverlays = (scene, button, style, title) =>
     button.overlays.set("caption", addText(scene, 0, 0, title, style).setOrigin(0.5));
 
 const resizeButton = pane => (button, idx) => {
-    const right = Boolean(pane.container.scene.config.menu.buttonsRight);
-    const bounds = pane.container.list[0].getBounds();
+    const right = Boolean(button.scene.config.menu.buttonsRight);
+    const bounds = pane.rect.getBounds();
```
Comment (AnthonyGoodchild) [inline src/components/shop/menu-buttons.js:47]: 👍

File: src/components/shop/scrollable-list/backgrounds.js [hunk 2]
```diff
@@ -5,7 +5,9 @@
  */
 const nullToUndefined = val => (val === null ? undefined : val);
 
-export const createBackground = {
+const getType = value => Object.prototype.toString.call(value).slice(8, -1).toLowerCase();
```
Comment (AnthonyGoodchild) [inline src/components/shop/scrollable-list/backgrounds.js:8]: >.slice(8, -1).toLowerCase();
is this ok? 😂

Comment (FostUK) [reply src/components/shop/scrollable-list/backgrounds.js:8]: Slack Overkill says so.

**Review by AnthonyGoodchild — APPROVED** [review]
nice work :heavy_check_mark:

**Pull request merged**